test/config.test.ts

```diff
@@ -64,6 +64,7 @@ function assertSchema(
             // skip full check if we are not encoding refs
             validateFormats: config.encodeRefs === false ? undefined : true,
             keywords: config.markdownDescription ? ["markdownDescription"] : undefined,
+            allowUnionTypes: true,
```

Comment:
Let's remove this so we see warnings until we figure out the right solution.

Reply:
done